src/tests/unittests/algorithms/test_functions.py

```diff
@@ -150,10 +150,28 @@ def test_fit_gauss_ramsey(self, fig=None):
 
         par_fit_test, results = fit_gauss_ramsey(x_data, y_data)
 
-        self.assertAlmostEqual(par_fit_test[0], 0.33, places=1)
-        self.assertAlmostEqual(par_fit_test[1] * 1e6, 3.36, places=1)
+        self.assertAlmostEqual(par_fit_test[0], 0.303, places=1)
+        self.assertAlmostEqual(par_fit_test[1] * 1e6, 3.55, places=1)
         self.assertAlmostEqual(par_fit_test[2] * 1e-6, 0.36, places=1)
 
+    def test_fit_gauss_ramsey_weights(self, fig=None):
+        x_data = np.hstack( (np.linspace(0, 3*np.pi, 20), np.linspace(3*np.pi, 4*np.pi, 100) ) )
```

Comment:
Maybe autopep this test.
I didn't get the plot, what can be the cause for that?

Reply:
The default argument `fig=None` means no plotting. I modified the test to plot by default. 

(the reason it is `None` by default is because of the split of tests into unittest and integration test, there is a ticket to revert this split)

Reply:
No I didn't mean that, sorry for the misunderstanding. I still think unit test should not show any output that is not part of the test.
Even when I add a number to fig, No plot is shown in my environment? But I think this shouldn't be part of the review comment because it is likely something in my environment.